assignments/prototype-refactor.js

```diff
@@ -1,41 +1,40 @@
-// Here we have a functioning solutoin to your challenge from yesterday.
+// Here we have a functioning solution to your challenge from yesterday.
 // Today your goal is to refactor all of this code to use ES6 Classes.
 // The console.log() statements should still return what is expected of them.
 
-function GameObject(options) {
-  this.createdAt = options.createdAt;
-  this.dimensions = options.dimensions;
-}
-
-GameObject.prototype.destroy = function() {
-  return `Object was removed from the game.`;
-};
+class GameObject {
+  constructor(options) {
+    this.createdAt = options.createdAt;
+    this.dimensions = options.dimensions;
+  }
+  destroy() {
+    return `Object was removed from the game.`;
+  }
```

Comment:
This is supposed to print the name of the object being removed, not the word `Object`.